scargo/commands/check.py

```diff
@@ -189,31 +189,33 @@ def check(self) -> None:
             return
         super().check()
 
+    def __cfile_copyrights(self) -> List[str]:
```

Comment:
Also, names of private methods should start with a single underscore.

Reply:
cfile means for source files in C language, because inside method body copyrights string is transformed on the list of the comments in C- type language.

Reply:
Renamed on __get_copyright_lines()